integration/container_restart_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
new files / existing files with new changes should be or add: "Copyright 2019 The containerd Authors."